internal/engine/buildcontroller_test.go

```diff
@@ -496,7 +496,7 @@ func TestCrashRebuildTwoContainersTwoImages(t *testing.T) {
 	defer f.TearDown()
 
 	manifest := manifestbuilder.New(f, "sancho").
-		WithK8sYAML(testyaml.SanchoTwoContainersOneImageYAML).
```

Comment:
This test is `TestCrashRebuildTwoContainersTwoImages` but was using _one_ image YAML